app/controllers/api/v1/participant_declarations_controller.rb

```diff
@@ -6,7 +6,7 @@ class ParticipantDeclarationsController < Api::ApiController
       include ApiTokenAuthenticatable
 
       def create
-        params = HashWithIndifferentAccess.new({ raw_event: request.raw_post, cpd_lead_provider: cpd_lead_provider }).merge(permitted_params["attributes"] || {})
+        params = HashWithIndifferentAccess.new({ raw_event: request.raw_post, lead_provider_from_token: cpd_lead_provider }).merge(permitted_params["attributes"] || {})
```

Comment:
Not sure I understand the name change.  I'd probably have changed the lead_provider_from_database rather than this one.

Reply:
I took it from
```
    def lead_provider_from_token
      params[:cpd_lead_provider]
    end
```

I think it explains what it is very well, and I am not sure this PR is the best place to change it.